sensor_msgs/NavSatFix[] nav_sat_data  # Array of satellite navigation data
float32[] x                           # Array of X coordinates
float32[] y                           # Array of Y coordinates
string[] type                         # Array of labels or identifiers
string[] colour                       # Array of color information
